packages/@aws-cdk/aws-ec2/test/vpc.test.ts

```diff
@@ -158,6 +166,33 @@ nodeunitShim({
       test.done();
     },
 
+    'with isolated and public subnet, should be able to use the internet gateway to define routes'(test: Test) {
```

Comment:
Add a test to validate the value of `internetGatewayId` in case all subnets are isolated.